ui/admin/tests/acceptance/sessions-test.js

```diff
@@ -73,74 +59,100 @@ module('Acceptance | sessions', function (hooks) {
       },
       'withAssociations'
     );
-    urls.orgScope = `/scopes/${instances.scopes.project.id}`;
-    urls.sessions = `/scopes/${instances.scopes.project.id}/sessions`;
+    urls.orgScope = `/scopes/${instances.scopes.org.id}/scopes`;
+    urls.projectScope = `/scopes/${instances.scopes.project.id}`;
+    urls.sessions = `${urls.projectScope}/sessions`;
+
+    authenticateSession({});
   });
 
   test('visiting sessions', async function (assert) {
     assert.expect(2);
-    authenticateSession({});
-    await visit(urls.sessions);
+    await visit(urls.projectScope);
+    await a11yAudit();
+
+    await click(`[href="${urls.sessions}"]`);
     await a11yAudit();
+
     assert.strictEqual(currentURL(), urls.sessions);
-    assert.strictEqual(findAll('tbody tr').length, instances.sessions.length);
+    assert.dom('tbody tr').exists({ count: instances.sessions.length });
   });
 
-  test('Users cannot navigate to sessions without proper authorization', async function (assert) {
+  test('users cannot navigate to sessions tab without proper authorization', async function (assert) {
     assert.expect(2);
-    instances.scopes.project.authorized_collection_actions.sessions = [];
-    authenticateSession({});
     await visit(urls.orgScope);
-    assert.notOk(
+    await a11yAudit();
+    instances.scopes.project.authorized_collection_actions.sessions =
+      instances.scopes.project.authorized_collection_actions.sessions.filter(
+        (item) => item !== 'list'
+      );
+
+    await click(`[href="${urls.projectScope}"]`);
+
+    assert.false(
       instances.scopes.project.authorized_collection_actions.sessions.includes(
         'list'
       )
     );
-    assert.notOk(find(`[href="${urls.sessions}"]`));
+    assert.dom(`[href="${urls.sessions}"]`).doesNotExist();
   });
 
-  test('Users can navigate to sessions with proper authorization', async function (assert) {
+  test('users can navigate to sessions with proper authorization', async function (assert) {
     assert.expect(2);
-    instances.scopes.project.authorized_collection_actions.sessions = ['list'];
-    authenticateSession({});
     await visit(urls.orgScope);
-    assert.ok(
+
+    await click(`[href="${urls.projectScope}"]`);
+
+    assert.true(
       instances.scopes.project.authorized_collection_actions.sessions.includes(
         'list'
       )
     );
-    assert.ok(find(`[href="${urls.sessions}"]`));
+    assert.dom(`[href="${urls.sessions}"]`).exists();
   });
 
   test('visiting sessions without users or targets is OK', async function (assert) {
-    assert.expect(2);
+    assert.expect(1);
+    await visit(urls.projectScope);
     instances.sessions[0].update({
       userId: null,
       targetId: null,
     });
-    authenticateSession({});
```

Comment:
Good catch removing the `authenticateSession` from each test and putting it within `beforeEach` :partying_face: